doc/reflection-milestone4.md

```diff
@@ -0,0 +1,68 @@
+# Milestone 4 Reflection: Safe Vancity Dashboard App
+
+> Authors: Arlin Cherian, Victor Francis, Wanying Ye
+
+## 1. What we have implemented
+
+We have made most of the implementations that we outlined in our proposal. These implementations include:
+
+1.  An interactive map of `Crime Density by Neighbourhood, Crime Type, and  Month`;
+
+2.  An interactive bar chart of `Total Reported Cases by Neighbourhood and Crime Type`;
+
```

Comment:
You can also add a line here that says, the improvements made to the py-version of the dashboard is also implemented in R.

Reply:
Also, we added summary stats for all of Vancouver - you can write about that as well